### DISCUSSION TITLE
用Excel進行數據分析——分析股票行情

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS3073-C/Python-Community-GS3073-C#472] [科技新知]
這裡介紹了如何用Excel對股票進行數據分析，屬於比較進階的技術，大家可以參考看看，文章中也有提到Tushare這個免費、開源的python財經數據介面包。主要實現對股票等金融數據從數據採集、清洗加工 到 數據存儲的過程，能夠為金融分析人員提供快速、整潔、和多樣的便於分析的數據，為他們在數據獲取方面極大地減輕工作量，使他們更加專註於策略和模型的研究與實現上。考慮到Python pandas包在金融量化分析中體現出的優勢，Tushare返回的絕大部分的數據格式都是pandas DataFrame類型，非常便於用pandas/NumPy/Matplotlib進行數據分析和可視化。

我覺得要用AI來預測股價顯然不太可行，但是在量化交易、提高勝率、預測行情的部分還是蠻可靠的一個方法。

文章網址：
https://www.getit01.com/p2018081540197112/

**Comment by manipulateed** [comment]
想不到用EXCEL就可以數據分析，算是進入用PYTHON的套件分析前的試煉，感謝分享。

**Comment by lx3211** [comment]
利用excel的GUI介面來進行資料處理，感覺很適合作為入門，了解步驟

**Comment by Chenszuhong** [comment]
感覺這很適合初學者的第一次嘗試，感謝分享

**Comment by limitx0** [comment]
原來 Excel 也能做到這些
每次修改能直接看到結果，感覺在入門時更容易學習

**Comment by Ben6uo** [comment]
看來從今天開始要向財富自由走了

**Comment by magicmicheal** [comment]
最近上其他通識課時，聽到有人想作用AI預測股市，感覺也是個十分有趣的想法，只不過馬上被老師打槍，我認為從Excel這個方法來嘗試，應該是個不錯的開始

**Comment by bww0401** [comment]
難道這就是財富密碼嗎，雖然我感覺預測股市向來都是天方夜譚，但是excel能夠幫助人類分析股市是一個很好的開始。跟著excel走，財富都到手: )
